### ISSUE TITLE
Multiple Handles (more than two)

### ISSUE THREAD

**Issue by vicatcu** [angular-slider/ng5-slider#115]
I would be interested in having the ability to have multiple handles specified. It would be a convenient UI for partitioning a range (represented by the full slider length) into multiple contiguous ranges. The bound model value could be an array of values in this case, e.g. `[a,b,c]`. Another alternative for the bound model value would be an array of 2-valued range arrays, e.g. `[ [a,b] , [b,c] ]`. The latter opens of the possibility of specifying multiple disjoint ranges, which could be kind of interesting.

Is this already possible? Has it been considered before? Am I off my rocker?

**Comment by piotrdz** [comment]
No, sorry, there is no way of doing this currently, and adding support for more handles is going to be tricky. It would be almost like re-inventing the slider from scratch.

I may go back to this at some point, but for the present, I am focusing on current slider features.

**Comment by phdesjardins** [comment]
Would like to contribute to creating this feature.
